app/proguard-rules.pro

```diff
@@ -84,3 +84,10 @@
 # A reference class used by androidx.concurrent.futures.AbstractFuture$UnsafeAtomicHelper
 # Currently consumed (in depenendecies) by mozilla.components.service.freboard.scheduler
 -dontwarn sun.misc.Unsafe
+
+####################################################################################################
+# Glean SDK
+####################################################################################################
+
+# Keep code generated from Glean Metrics
+-keep class org.mozilla.tv.firefox.GleanMetrics.** { *; }
```

Comment:
This may not be needed anymore since the introspection was removed from Glean.  *If* it's not a distraction, it might be worth experimenting with taking this out.

Reply:
I was thinking that may have been the case but I was just being sure.  This _does_ still exist in Fenix currently.

Reply:
I agree with @mdboom , let's remove this and make sure to try a *final build* (the release one generated on CI for which proguard applies) on a device to make sure. I'm  confident it will work :-)

Reply:
Yeah.  We need to go back and remove it from Fenix, too.  I got a bit blocked by duplicating the problem locally, and I didn't want to push the change without testing locally first.